### DISCUSSION TITLE
Integration with Selenide framework

### DISCUSSION THREAD

**Discussion by svazham** [allure-framework/allure2#2109] [TestOps]
I have a testing framework with Selenide-testng-maven. 

1. How can I view/get my test automation results on the Allure Testops dashboard without having to manually click upload the results each time by going to allure-results- right click- Upload Test Results?
2. How can I see just only the test cases under that was run recently when I go to Tree section?
Currently I am seeing all the test cases run so far ever in Launches>Sample test run  (Refer Tree_TestResults.png)
![Tree_TestResults](https://github.com/allure-framework/allure2/assets/122938148/8a5bc7c9-889a-41c6-9a5f-6c5160bb608a)

**Comment by cheshi-mantu** [comment]
hi @svazham 
how on which CI system do you execute your tests?

here is Allure Testops docs pages reference for your information: https://docs.qameta.io/allure-testops/quickstart/upload/#test-result-upload-options
it describes all the ways to upload the test results to Allure Testops

**Reply by svazham** [reply]
Thanks-Currently I am not using any CI system to trigger tests- doing it from testNg.xml directly. Is there a way to uplaod without clicking Upload Test Results each time? Or is it possible only if CI is used ?

Also,
How can I see just only the test cases under that was run recently when I go to Tree section?
Currently I am seeing all the test cases run so far ever in Launches>Sample test run (Refer Tree_TestResults.png)

**Reply by cheshi-mantu** [reply]
pls refer to the documentation: https://docs.qameta.io/allure-testops/quickstart/upload/#test-result-upload-options
for the local runs, you can use allurectl as described here: https://docs.qameta.io/allure-testops/ecosystem/allurectl/

**Comment by svazham** [comment]
Thanks for sharing the documentation !
I'd like to know if using the non CI mode, would we still need to do the upload manually to view results in dashboard?

Also , I'm assuming there is no way to view just only the test cases under that was run recently when I go to Tree section?(Since I didnt receive a reply on this)
Currently I am seeing all the test cases run so far ever in Launches>Sample test run (Refer Tree_TestResults.png)

Thanks
